docs/guides/high-availability/operate-switch-over.adoc

```diff
@@ -0,0 +1,91 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Switch over to the secondary site"
+summary="This describes the operational procedures necessary"
+preview="true" >
+
+This procedure switches from the primary site to the secondary site when using a setup as outlined in <@links.ha id="concepts-active-passive-sync" /> together with the blueprints outlined in <@links.ha id="bblocks-active-passive-sync" />.
+
+include::partials/infinispan/infinispan-attributes.adoc[]
+
+== When to use this procedure
+
+Use this to take the primary offline gracefully.
+
+Once the primary site is back online, use the {sections} <@links.ha id="operate-network-partition-recovery" /> and <@links.ha id="operate-switch-back" /> to return to the original state with the primary site being active.
+
+See the <@links.ha id="introduction" /> {section} for different operational procedures.
+
+== Procedures
+
+=== {ispn} Cluster
+
+For the context of this {section}, `{site-a}` is the primary site and `{site-b}` is the secondary site.
+
+For taking a site offline, it is a good practice to disable the replication towards it.
+It prevents errors/delays when the channels are disconnected between the primary and the secondary site.
+
+==== Procedures to transfer state from secondary to primary site
+
+. Login into your secondary site
+
+include::partials/infinispan/infinispan-cli-connect.adoc[]
+
+. Disable the replication to the primary site by running the following command:
++
+Command:
+[source,bash,subs="+attributes"]
+----
+site take-offline --all-caches --site={site-a-cr}
+----
++
+Output:
+[source,bash,subs="+attributes"]
```

Comment:
Command/Output not rendered in bold, and there's a `+` shown before Output:
![image](https://github.com/keycloak/keycloak/assets/2271511/122d5e30-8244-4017-9147-5020ff44d86c)

Reply:
Thanks, I missed the leading dot to make "Command" and "Output" a title.